mac_build.sh

```diff
@@ -80,17 +90,22 @@ export VORBISENC_CFLAGS="-I$(pwd)/libvorbis-$VORBISVERSION/include"
 export VORBISENC_LIBS="$(pwd)/libvorbis-$VORBISVERSION/lib/libvorbisenc.la"
 export OPUS_CFLAGS="-I$(pwd)/opus-$OPUSVERSION/include"
 export OPUS_LIBS="$(pwd)/opus-$OPUSVERSION/libopus.la"
-export LAME_CFLAGS="-I$(pwd)/lame-$LAMEVERSION/include"
-export LAME_LIBS="$(pwd)/lame-$LAMEVERSION/libmp3lame/.libs/libmp3lame.la"
-export MPG123_CFLAGS="-I$(pwd)/mpg123-$MPG123VERSION/src/libmpg123 $LAME_CFLAGS"
-export MPG123_LIBS="$(pwd)/mpg123-$MPG123VERSION/src/libmpg123/libmpg123.la $LAME_LIBS"
+export MPEG_CFLAGS="-I$(pwd)/lame-$LAMEVERSION/include"
+export MPEG_LIBS="$(pwd)/lame-$LAMEVERSION/libmp3lame/.libs/libmp3lame.la"
+export MPG123_CFLAGS="-I$(pwd)/mpg123-$MPG123VERSION/src/libmpg123 $MPEG_CFLAGS"
+export MPG123_LIBS="$(pwd)/mpg123-$MPG123VERSION/src/libmpg123/libmpg123.la $MPEG_LIBS"
 
 curl -LO https://github.com/libsndfile/libsndfile/releases/download/$SNDFILE_VERSION/libsndfile-$SNDFILE_VERSION.tar.xz
 tar jxvf libsndfile-$SNDFILE_VERSION.tar.xz
 cd $SNDFILENAME
-./configure --disable-static --disable-sqlite --disable-alsa
+./configure $BUILD_HOST --disable-static --disable-sqlite --disable-alsa --disable-full-suite
```

Comment:
This is missing `CFLAGS=$EXTRA_CFLAGS CXXFLAGS=$EXTRA_CFLAGS`, I think.

Reply:
I think when you add these CFLAGS it back, it fails to find the `lame` library again